### PR TITLE
add showcase for xiaoxian

### PR DESCRIPTION
**Pull request by rlog**
RT

### CODE REVIEW

**Comment by dsibiski** [comment]
@mkonicek What's the thoughts on a Showcase link that goes to a page with just a link to an APK?

/cc @brentvatne

**Comment by chirag04** [comment]
IMO, it's fine as long as it's a valid app download/info link. some people may want to do analytics based on a custom link.

**Comment by facebook-github-bot-0** [comment]
@rlog updated the pull request.

**Comment by dsibiski** [comment]
Thanks @chirag04, good point!

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -205,6 +205,12 @@ var featured = [
     linkPlayStore: 'https://play.google.com/store/apps/details?id=me.ele.crowdsource',
     author: 'Eleme',
   },
+  {
+    name: '饿小闲',
+    icon: 'https://xiaoxian.ele.me/download/images/share_page/ic_launcher.png',
+    link: 'https://xiaoxian.ele.me/download/?td_channelid=RNshowcase',
+    author: 'Eleme',
+  },
 ];
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: @rlog Can you please move this down to the "Apps" section, since the "Featured" section is reserved. Thanks!

**Comment by facebook-github-bot-1** [comment]
@rlog updated the pull request.

File: website/src/react-native/showcase.js (hunk 1 above)
Comment (rlog) [inline website/src/react-native/showcase.js]: Done, Thanks for your review  :)

**Comment by facebook-github-bot-2** [comment]
@rlog updated the pull request.

**Comment by rlog** [comment]
💔

File: website/src/react-native/showcase.js [hunk 2]
```diff
@@ -917,6 +917,12 @@ var apps = [
     icon: 'https://lh3.googleusercontent.com/H0SILVHcDUxSMoSQwMb2QYtLjTBCqvK5ZEjOEwKQQ-2qnRV6Hd9Hn-gtSGPaoIOPwA=w300-rw',
     link: 'https://play.google.com/store/apps/details?id=com.biideal.biichat',
     author: 'biideal Inc.'
+  },
+  {
+    name: '饿小闲',
+    icon: 'https://xiaoxian.ele.me/download/images/share_page/ic_launcher.png',
+    link: 'https://xiaoxian.ele.me/download/?td_channelid=RNshowcase',
+    author: 'Eleme',
   }
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: Sorry, one more nit pick, please add a trailing comma here.

**Comment by facebook-github-bot-5** [comment]
@rlog updated the pull request.

File: website/src/react-native/showcase.js (hunk 2 above)
Comment (rlog) [inline website/src/react-native/showcase.js]: done

**Comment by facebook-github-bot-5** [comment]
@rlog updated the pull request.

File: website/src/react-native/showcase.js [hunk 3]
```diff
@@ -970,7 +970,13 @@ var apps = [
     icon: 'https://lh3.googleusercontent.com/H0SILVHcDUxSMoSQwMb2QYtLjTBCqvK5ZEjOEwKQQ-2qnRV6Hd9Hn-gtSGPaoIOPwA=w300-rw',
     link: 'https://play.google.com/store/apps/details?id=com.biideal.biichat',
     author: 'biideal Inc.'
-  }
+  },
+  {
+    name: '饿小闲',
+    icon: 'https://xiaoxian.ele.me/download/images/share_page/ic_launcher.png',
+    linkAppStore: 'https://itunes.apple.com/cn/app/e-xiao-xian/id1092025196',
+    author: 'Eleme'
```
Comment (dsibiski) [inline website/src/react-native/showcase.js]: You had a comma here before and now you removed it! haha 😄 

Please add it back as every object/array should have trailing commas as per the style guide. https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#general

**Comment by facebook-github-bot-3** [comment]
@rlog updated the pull request.

**Comment by facebook-github-bot-9** [comment]
@brentvatne would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1679919248937846/int_phab) to review.

**Comment by qingfeng** [comment]
@rlog cool

**Pull request closed without merging**